win/tty/termcap.c

```diff
@@ -737,52 +735,10 @@ static const short tmspc10[] = { /* from termcap */
 #endif
 #endif
 
-/* delay 50 ms */
+/* NLE: Don't delay ever. */
```

Comment:
Need for speed?

Reply:
Yes (although we didn't call this function); but also this includes symbols that aren't present if `TERMLIB` isn't defined, and doesn't properly protect against that with `#ifdef`s.